tests/integration/components/bs-tab-test.js

```diff
@@ -142,12 +142,45 @@ test('tab navigation is groupable', function(assert) {
   assert.equal(find('ul.nav.nav-tabs > li:nth-child(2) > a').textContent.trim(), 'Tab 2', 'navigation item shows pane title');
   assert.equal(find('ul.nav.nav-tabs > li:nth-child(3)').classList.contains('dropdown'), true, 'adds dropdown for grouped items');
   assert.equal(find('ul.nav.nav-tabs > li:nth-child(3) > a').textContent.trim(), 'Dropdown', 'drop down item shows pane groupTitle');
-  assert.equal(findAll('ul.nav.nav-tabs > li:nth-child(3) > ul.dropdown-menu > li').length, 2, 'puts items with groupTitle under dropdown menu');
+  assert.equal(find('ul.nav.nav-tabs > li:nth-child(3) > .dropdown-menu').children.length, 2, 'puts items with groupTitle under dropdown menu');
   assert.equal(find(
-    'ul.nav.nav-tabs > li:nth-child(3) > ul.dropdown-menu > li:nth-child(1) > a'
+    'ul.nav.nav-tabs > li:nth-child(3) > .dropdown-menu > :nth-child(1) > a'
   ).textContent.trim(), 'Tab 3', 'dropdown menu item shows pane title');
   assert.equal(find(
-    'ul.nav.nav-tabs > li:nth-child(3) > ul.dropdown-menu > li:nth-child(2) > a'
+    'ul.nav.nav-tabs > li:nth-child(3) > .dropdown-menu > :nth-child(2) > a'
```

Comment:
If you omit the last part of the selector (`> a`) here and on the statement before, this should still work, doesn't it? This is the only difference between the BS3 and BS4 tests, or did I miss something else? This way we could maybe unify the tests to avoid too much duplication?